files/passwd

```diff
@@ -57,3 +57,4 @@ opensaf:x:342::::
 # free space
 radiusd:x:301::::
 postgres:x:28::::
+arpwatch:x:402::::
```

Comment:
Could you move this line up so that the UID/GIDs are sorted in reverse order?

Reply:
I have updated the commit summary line and have also moved arpwatch UID/GIDs up in a proper order.